README.md

```diff
@@ -3,10 +3,10 @@
 ### Goal
 The goal is to find the most profitable settings for the DOMINUS strategy.
 ### Scope
-The current DOMINUS version (v2) has 10 different variables that affect entries and exits (not including the 9 variables for the ADX and the BB). We built an automation that iterates though the permutations of those settings and saves the results in a `.csv` file for data analysis.
+The current DOMINUS version (v2) has 10 different variables that affect entries and exits (not including the 9 variables for the ADX and the BB). We built an automation that iterates though the permutations of those settings and save the results in a `.csv` file for data analysis.
```

Comment:
You had me here for a moment. I was going for "the automation saves ..." but you went for "We built ... and save ...".
Both are fine by me.